dissect/volume/vinum/config.py

```diff
@@ -0,0 +1,407 @@
+from __future__ import annotations
+
+import logging
+import os
+import re
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum, auto
+from typing import TypedDict
+
+from dissect.volume.vinum.c_vinum import c_vinum
+
+log = logging.getLogger(__name__)
+log.setLevel(os.getenv("DISSECT_LOG_VINUM", "CRITICAL"))
+log.addHandler(logging.StreamHandler())
+
+
+@dataclass
+class Volume:
+    tstamp: datetime
+    name: bytes
```

Comment:
Decode to `str`?

Reply:
No, that is not guaranteed to be meaningful, better keep it close to the source.